lib/resty/auto-ssl/init_master.lua

```diff
@@ -82,6 +89,10 @@ local function setup_storage(auto_ssl_instance)
 end
 
 return function(auto_ssl_instance)
+  if not ngx.shared.auto_ssl_settings then
+      ngx.log(ngx.ERR, "auto-ssl: dict auto_ssl_settings could not be found. Please add it to your configuration: `lua_shared_dict auto_ssl_settings 1m;`")
```

Comment:
Nice! This should help with upgrades. I'd just make sure this size in this recommendation aligns with whatever the README says (which I had proposed lowering to 64k).